src/Mailcode/Translator/Syntax/ApacheVelocity/ShowSnippetTranslation.php

```diff
@@ -31,21 +31,17 @@ public function translate(Mailcode_Commands_Command_ShowSnippet $command): strin
     {
         $varName = undollarize($command->getVariableName());
 
-        if($command->isHTMLEnabled())
-        {
-            $statement = sprintf(
-                '%s.replaceAll($esc.newline, "<br/>")',
-                $varName
-            );
-        }
-        else
-        {
-            $statement = sprintf(
-                '%s',
-                $varName
-            );
+        if ($command->isNamespacePresent()) {
+            $namespace = $command->getNamespaceToken()->getText();
+            $statement = sprintf('dictionary.namespace("%s").name("%s")', $namespace, $varName);
+        } else {
+            $statement = sprintf('dictionary.global("%s")', $varName);
         }
 
+        $statement = $command->isHTMLEnabled()
+            ? sprintf('%s.replaceAll($esc.newline, "<br/>")', $statement)
```

Comment:
I would really prefer if you could change this to a regular IF syntax. I never use the shorthand because it always requires that one brain cell more 😅